### PR TITLE
Behavior of dapr uninstall -k and dapr uninstall is different when dapr not installed

### PR DESCRIPTION
**Pull request by hueifeng**
# Description

Signed-off-by: HueiFeng <695979933@qq.com>

## Issue reference

#952 

Please reference the issue this PR will close: #952 

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [x] Code compiles correctly
* [ ] Created/updated tests
* [ ] Extended the documentation

### CODE REVIEW

**Comment by hueifeng** [comment]
Now they output ` WARNING ` log:
```
PS I:\source\cli> ./cli.exe uninstall -k
Removing Dapr from your cluster...
WARNING: dapr release does not exist
Dapr has been removed successfully
```

cc @mukundansundar

**Comment by mukundansundar** [comment]
@hueifeng Can you add an e2e test for the same?

**Comment by hueifeng** [comment]
@mukundansundar Let me try it.

**Comment by hueifeng** [comment]
@mukundansundar ping

File: pkg/kubernetes/uninstall.go [hunk 1]
```diff
@@ -29,10 +29,20 @@ func Uninstall(namespace string, uninstallAll bool, timeout uint) error {
 	if err != nil {
 		return err
 	}
+	exists, err := confirmExist(config)
+	if err != nil {
+		return err
+	}
+
+	if !exists {
+		print.WarningStatusEvent(os.Stdout, "WARNING: %s release does not exist", daprReleaseName)
```
Comment (mukundansundar) [inline pkg/kubernetes/uninstall.go:38]: Print warning to `os.StdErr`

**Comment by mukundansundar** [comment]
@hueifeng there are commits in this PR unrelated to this PR... Can you fix that?

**Comment by mukundansundar** [comment]
> @hueifeng there are commits in this PR unrelated to this PR... Can you fix that?

@hueifeng i can see that the commits are fine but now all tests are failing. Can you check that?

**Comment by hueifeng** [comment]
@mukundansundar It looks a lot now.

**Review by pravinpushkar — APPROVED** [review]
LGTM

**Pull request merged**